features/routing/haproxy-router.feature

```diff
@@ -352,12 +347,17 @@ Feature: Testing haproxy router
       | /tmp/cookies |
     Then the step should succeed
     And the output should contain "Hello-OpenShift"
+    Given I run the steps 10 times:
+    """
     When I execute on the pod:
-      | bash | -c | for i in {1..10} ; do curl -sS  http://<%= route("service-unsecure", service("service-unsecure")).dns(by: user) %>/ -b /tmp/cookies ; done |
+      | curl |
+      | -sS |
+      | http://<%= route("service-unsecure", service("service-unsecure")).dns(by: user) %>/ |
+      | -b |
+      | /tmp/cookies |
     Then the step should succeed
-    And the output should contain:
-      | Hello-OpenShift web-server-1 |
-      | Hello-OpenShift web-server-2 |
```

Comment:
please keep the original checkpoint as well.

Reply:
Yes, I have kept the original checkpoint. Just remove the test-client-pod and add pod-for-ping before this checkpoint, thanks.